website/docs/docs/platform/2-writing-your-first-test.md

```diff
@@ -40,7 +44,41 @@ export default test('bigtest todomvc')
   );
 ```
 
-Some things to know:
+For your first test, you can start with something small, and then build it up after you see it running successfully. Let's assert that an app's heading text has rendered. 
+
+First, look for some text on your app's index page that is wrapped in an `<h1>`.
+
+Then, copy and paste the example below into your own test file, substituting "My Heading Text" in the example below with your own text.
+
+```js
+import { Heading, Page, test } from `bigtest`;
+
+export default test('home page rendering')
+  .step(Page.visit('/'))
+  .assertion(Heading('My Heading Text').exists())
+```
+
+If your app does not have any heading text, you could check for a button instead:
+
+```js
+import { Button, Page, test } from `bigtest`;
+
+export default test('home page rendering')
+  .step(Page.visit('/'))
+  .assertion(Button('My Button Text').exists())
+```
+
+## Interactors
+
+In the examples above, we used Interactors like `Button` and `Heading` to locate something in the UI, interact with it like a user would, and make a test assertion. Learn more about how interactors work in our [Interactors](../interactors) guides.
+
+## Test order
+
+If you are familiar with other testing libraries, might have a few questions at this point about testing order. Here is some basic information to help you connect the dots for how BigTest works.
+
```

Comment:
I really like how this is introduced. Can we add some more simple examples? We haven't seen `children` yet for instance.

Reply:
the children example is in between lines 35-44